lib/default.nix

```diff
@@ -393,4 +393,96 @@ in {
   # one in the list will be used.
   combineFlakes = sep: prefixAndFlakes: builtins.foldl' addFlakes {}
     (lib.mapAttrsToList (prefix: flake: prefixFlake prefix sep flake) prefixAndFlakes);
+
+  mkFlake = project: packages: coverageProjectModule:
```

Comment:
`packages` argument seems dead?

Reply:
Renamed `selectPackages` and it is now used.